lib/languageFeatures/findObjectFromPosition.ts

```diff
@@ -0,0 +1,18 @@
+import { Position } from "vscode-languageserver";
+import { ObjectBase } from "../parser/objects";
+import { VhdlLinter } from "../vhdl-linter";
+
+export function findObjectFromPosition(linter: VhdlLinter, position: Position): ObjectBase[] {
+  const startI = linter.getIFromPosition(position);
+  let candidates = linter.file?.objectList.filter(object => object.range.start.i <= startI && startI <= object.range.end.i) ?? [];
+  for (const architecture of linter.file.architectures) {
+    candidates.push(...architecture.components.filter(object => object.range.start.i <= startI && startI <= object.range.end.i));
+  }
```

Comment:
what is this for? Are components not in the objectList? And why only in design unit architectures?